packages/j/jsxgraph.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "jsxgraph",
+  "filename": "jsxgraphcore.js",
+  "description": "A JavaScript library for interactive geometry, function plotting, visualization",
+  "keywords": [
+    "dynamic geometry",
+    "function plotting",
+    "mathematics education",
+    "geometry",
+    "calculus"
+  ],
+  "homepage": "https://jsxgraph.org/",
+  "author": {
+    "name": "Alfred Wassermann",
+    "email": "alfred.wassermann@uni-bayreuth.de",
+    "url": "https://www.dmi.uni-bayreuth.de/de/team/wassermann/"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/jsxgraph/jsxgraph.git"
+  },
+  "npmName": "jsxgraph",
+  "npmFileMap": [
+    {
+      "basePath": "distrib",
+      "files": [
+        "**/*.+(js|css)"
```

Comment:
This is going to include a lot of assets from the docs, which we don't want?

Reply:
Sorry, I will postpone that after cleaning the npm repository with the next release.

Reply:
We will still pull old versions though, using this fileMap, so this will still match assorted docs files from previous versions. Can we update the fileMap so it only matches the required files please?

Reply:
I recommend using https://www.digitalocean.com/community/tools/glob and importing the NPM package, so you can see what the glob would match.